keel-core/src/main/kotlin/com/netflix/spinnaker/keel/attribute/attributes.kt

```diff
@@ -36,3 +36,22 @@ class PriorityAttribute(value: IntentPriority) : Attribute<IntentPriority>("Prio
 
 @JsonTypeName("Enabled")
 class EnabledAttribute(value: Boolean) : Attribute<Boolean>("Enabled", value)
+
+@JsonTypeName("ExecutionWindow")
+class ExecutionWindowAttribute(value: ExecutionWindow) : Attribute<ExecutionWindow>("ExecutionWindow", value) {
```

Comment:
I think I might need to rethink the `Attribute` interface a little bit. It'd be way nicer to have something more like:

```
class ExecutionWindowAttribute(days: List<Int>, timeWindows: List<TimeWindow>) ...
```